diff --git a/lib/mirage.ml b/lib/mirage.ml
index f3bbf27..e9061a6 100644
--- a/lib/mirage.ml
+++ b/lib/mirage.ml
@@ -662,8 +662,9 @@ let compile ignore_dirs libs warn_error target =
   and lflags =
     let dontlink =
       match target with
-      | `Xen | `Qubes | `Virtio | `Hvt | `Muen | `Genode | `Riscv ->
+      | `Xen | `Qubes | `Virtio | `Hvt | `Muen | `Genode ->
         ["unix"; "str"; "num"; "threads"]
+      | `Riscv -> ["unix"; "str"; "threads"]
       | `Unix | `MacOSX -> []
     in
     let dont = List.map (fun k -> [ "-dontlink" ; k ]) dontlink in
